engines/tika/src/test/java/org/alfresco/transform/tika/TikaMetadataExtractsIT.java

```diff
@@ -24,56 +24,58 @@
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_IWORK_PAGES;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_MP3;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_FORMULA;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_GRAPHICS;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_GRAPHICS_TEMPLATE;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_PRESENTATION;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_PRESENTATION_TEMPLATE;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_SPREADSHEET;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_SPREADSHEET_TEMPLATE;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_TEXT;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_TEXT_TEMPLATE;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENOFFICE1_WRITER;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_PRESENTATION;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_SPREADSHEET;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_WORDPROCESSING;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_PDF;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_PPT;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_TEXT_PLAIN;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_VIDEO_3GP;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_VIDEO_3GP2;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_VIDEO_FLV;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_VIDEO_MP4;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_VIDEO_QUICKTIME;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_VISIO;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_VORBIS;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_WORD;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_XML;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_ZIP;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_IMAGE_RAW_RAF;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_IMAGE_RAW_ARW;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_IMAGE_RAW_CR2;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_IMAGE_RAW_RW2;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_IMAGE_RAW_NEF;
+import static org.alfresco.transform.common.Mimetype.MIMETYPE_APP_DWG;
+import static org.alfresco.transform.common.Mimetype.MIMETYPE_OUTLOOK_MSG;
+import static org.alfresco.transform.base.clients.FileInfo.testFile;
```

Comment:
Import not used. All calls to this method have been accessed: `FileInfo.testFile()`.
Remove import or remove `FileInfo.` from the usage of `testFile()` (The latter would be preferred as it would reduce the diff)

Reply:
Cleaned up. Using static import of testFile. Looks like an automatic refactor, as there were commented out usages.